main.c

```diff
@@ -313,6 +318,11 @@ int main(int argc, char **argv)
 					stb_type = BRCM7401;
 					break;
 				}
+				else if (strcasestr(buf,"DM900")) || (strcasestr(buf,"DM920"))
```

Comment:
It has to be:
else if (strcasestr(buf,"DM900") || strcasestr(buf,"DM920"))
Please close this merge I'll send a new one ...